SharpPcap/Npcap/NpcapDevice.cs

```diff
@@ -121,26 +119,6 @@ override protected void SendPacketArrivalEvent(RawCapture p)
             }
         }
 
-        /// <value>
-        /// Set the kernel value buffer size in bytes
-        /// Npcap extension
-        /// </value>
-        public override uint KernelBufferSize
-        {
-            set
-            {
-                ThrowIfNotNpcap();
```

Comment:
@chmorgan something does not add up with `KernelBufferSize` being a property.

According to https://linux.die.net/man/3/pcap_set_buffer_size `pcap_set_buffer_size` will fail with `PCAP_ERROR_ACTIVATED ` for an activated device, and `Open()` automatically activates the device.

Or am I missing something?

Reply:
It might have been that for npcap it could be set at runtime? We should follow what the current libpcap api is going now though, its been so long since I wrote this that I can't recall.

Reply:
We got argument `kernel_buffer_size` in `Open()` could we remove the property to avoid confusion?

Reply:
yeah lets remove the property if it can only be modified when we open/activate the device up.

Reply:
@chmorgan after more tests and reading libpcap source code, its seems `pcap_set_buffer_size` is NOT the same as `pcap_setbuff`:
* `pcap_set_buffer_size`: must be called before activating the device, and is for setting `user` buffer size.
* `pcap_setbuff`: must be called after activating the device, and is for setting `kernel` buffer size, available only in Windows.

See `pcap-npf.c`
```c
/*dimension of the buffer in the pcap_t structure*/
#define	WIN32_DEFAULT_USER_BUFFER_SIZE 256000

/*dimension of the buffer in the kernel driver NPF */
#define	WIN32_DEFAULT_KERNEL_BUFFER_SIZE 1000000
```
I beleive misxing up those two different concepts, is the root cause of #172 since setting kernel buffer size is broken due to the code trying to run before activating the device.

Reply:
Should we add yet another argument `buffer_size`?

Reply:
Ahh. So the open argument is buffer size and this property is kernel buffer size? Maybe we should rename the argument to open to remove kernel from its name and use pcap_set_buffer_size there?

Reply:
The problem is that existing code was treating them as if they were the same thing (No thanks to libpcap docs there).
This was causing a lot of mix-ups, hopefully the new code clear things a bit.